client/lib/sites-list/list.js

```diff
@@ -464,11 +465,16 @@ SitesList.prototype.getSelectedOrAllJetpackCanManage = function() {
 };
 
 SitesList.prototype.getSelectedOrAllWithPlugins = function() {
-	return this.getSelectedOrAll().concat(
+	return unique( this.getSelectedOrAll().concat(
```

Comment:
I don't understand what's happening here. It looks like we are concatenating a filtered list onto itself, then taking the unique values. Is the purpose simply to have the opposite set from `getSelectedOrWPComBusiness()`?

If that is the case, this would seem a complicated route to get there.

```js
import negate from 'lodash/function/negate';

const isBusinessSitePlan = site => isBusiness( site.plan );

StatesList.prototype.getSelectedOrAllWithPlugins = () => (
	this
		.getSelectedOrAll()
		.filter( negate( isBusinessSitePlan ) );
);
```